src/material-table.js

```diff
@@ -480,6 +480,8 @@ export default class MaterialTable extends React.Component {
 
   render() {
     const props = this.getProps();
+    const hasAnyEditingRow= !!(this.state.lastEditingRow || this.state.showAddRow);
+    !!props.onEditingStatusChange && props.onEditingStatusChange(hasAnyEditingRow);
```

Comment:
Calling an event on render function does not make sense? You can handle this change on material-table.js